src/Education/XamlGenerator/SubTitleImageInfo.cs

```diff
@@ -31,19 +31,19 @@ public class SubTitleImageInfo
     {
         public static readonly IReadOnlyDictionary<RuleIssueSeverity, SubTitleImageInfo> SeverityImages = new Dictionary<RuleIssueSeverity, SubTitleImageInfo>()
         {
-            { RuleIssueSeverity.Critical, new SubTitleImageInfo("/SonarLint.VisualStudio.Education;component/XamlGenerator/images/severity/critical.svg", "Critical" )},
```

Comment:
Previously the files were referenced directly. Now we need to reference the resource name in the ResourceDictionary.

Reply:
After this change the field `ImageResourcePath` and it's summary does not make sense. Maybe we should change it to `ImageResourceName` and change the summary to reflect the changes?

Reply:
Done.